bika/lims/content/worksheet.py

```diff
@@ -1535,12 +1546,11 @@ def copy_src_fields_to_dst(src, dst):
         new_ws_analyses = []
         old_ws_analyses = []
         for analysis in analyses:
+            position = analysis_positions[analysis.UID()]
```

Comment:
nice catch, but does not belong to this PR. Please remove that and make a new one. Same for the `type` vs `typ`